sentinel-demo/sentinel-demo-log-logback/pom.xml

```diff
@@ -12,12 +12,16 @@
     <artifactId>sentinel-demo-log-logback</artifactId>
 
     <dependencies>
+        <dependency>
+            <groupId>com.alibaba.csp</groupId>
+            <artifactId>sentinel-transport-common</artifactId>
+        </dependency>
+
```

Comment:
The dependency seems not used in demo, it is redundant

Reply:
Actually I'm planning to move the `CommandCenterLog` to `sentinel-transport-common`, as it's only used in transport command centers. This will be done in another PR, but the dependency could be reserved :)

Reply:
OK, got it. Now LogBase is not a parent class, maybe another class name is better, like LogConfig or sth else. Besides, I found that the config parameter which can be customized by user is scattered in different classes(by global seaching "csp.sentinel." in sentinel-core module), if it is better put them into one config class, to make config clearly and convenient to found. And this need careful consideration and discussion, and I think maybe we can do it in the future.

Reply:
Good idea. We may improve this later, but be careful of the compatibility. We could create an umbrella issue to discuss it :)